build-container/ccm.conf

```diff
@@ -0,0 +1,71 @@
+# Fully qualified path for build root.
+# This should not use a variable like $HOME.  If your machine has lots
+# of memory, consider locating this to tmpfs to avoid usage to the disk and
+# to minimize access times but know that unless you copy the contents to
+# physical media, it will not survive a reboot. See the manpage for tips.
+CHROOTPATH64="/scratch/.buildroot"
+
+# Number of threads makepkg in the clean chroot will use when building.
+# The typical rule is physical cores + 1.
+THREADS=9
```

Comment:
If this isn't overridden somewhere else, maybe we could adjust it with something like this from either `build`/`release.yml` or `entrypoint.sh`?
```sh
NUMPROC="$(awk '/cpu cores/ { print $4; exit}' /proc/cpuinfo)"
sed -i "/^THREADS=/s/9/${NUMPROC}/" /path/to/ccm.conf
```